### DISCUSSION TITLE
New NodeJS security releases

### DISCUSSION THREAD

**Discussion by viceice** [yao-pkg/pkg#53] [Q&A] [answered]
https://nodejs.org/en/blog/vulnerability/april-2024-security-releases/

Please update node versions. 🤗

**Comment by robertsLando** [comment]
Unfortunately nodejs 18 patch doesn't apply cleanly and I have not time right now to manually do it,

Node 20 patch is merged: https://github.com/yao-pkg/pkg-fetch/commit/9cb16d361670b5bd5dfff6db27a20b003095f996

**Reply by viceice** [reply]
Thanks, node v20 is more important to me 🙃

**Comment by viceice** [comment]
Again security releases 😕 

https://nodejs.org/en/blog/vulnerability/july-2024-security-releases

**Reply by robertsLando** [reply]
I'm blocked here: https://github.com/yao-pkg/pkg-fetch/pull/37

**Comment by robertsLando** [comment, marked as answer]
5.13 has latest nodejs 18/20 version and 5.14 now has also support for nodejs 22.8 🎉
